contrib/python/src/python/pants/contrib/python/checks/checker/checker.py

```diff
@@ -77,7 +77,7 @@ def _get_nits(self, filename):
       for i, nit in enumerate(plugin_factory(python_file)):
```

Comment:
This PR fails because we must maintain Py2 compatibility for everything in `contrib/python/src/python/pants/contrib/python/checks/checker`. All the other code in `contrib/python` is fine to use Py3-only features like f-strings, but the `checker` folder is treated as a plugin that may be run with Python 2 or Python 3 depending on the linted target's interpreter constraints.